### DISCUSSION TITLE
Why are JWT tokens are passed in the request body?

### DISCUSSION THREAD

**Discussion by MikhailGordienk** [eclipse-edc/Connector#744] [Q&A] [answered]
Why JWT tokens are passed in the request body and not in the header as regular bearer tokens?

**Comment by jimmarino** [comment, marked as answer]
This is part of the current IDS specification. There is a new IDS REST API where (I believe) tokens will be passed as headers. It may make more sense to pass them as part of a defined envelope header due to the practical size restrictions on HTTP headers.  

Support for the IDS REST API is currently in progress.

Alternatively, someone could create their own REST API for the connector.
